modules/installation-osp-kuryr-port-pools.adoc

```diff
@@ -17,7 +17,7 @@ Because Kuryr keeps each namespace in a separate subnet, a separate ports pool i
 
 Prior to installing a cluster, you can set the following parameters in the `cluster-network-03-config.yml` manifest file to configure ports pool behavior:
 
-* The `enablePortPoolsPrepopulation` parameter controls pool prepopulation, which forces Kuryr to add ports to the pool when it is created, such as when a new host is added, or a new namespace is created. The default value is `false`.
+* The `enablePortPoolsPrepopulation` parameter controls pool prepopulation, which forces Kuryr to add ports to the pools when the first pod on pods Network is created in a namespace. The default value is `false`.
```

Comment:
@MaysaMacedo To be clear, which interpretation best matches this clause: "the first pod on pods Network"

1. The first pod on the network for pods
2. The first pod on the network resource for pods
3. The first pod on the network that a pod belongs to

Reply:
```suggestion
* The `enablePortPoolsPrepopulation` parameter controls pool prepopulation, which forces Kuryr to add ports to pools when the first pod on the network for pods is created in a namespace. The default value is `false`.
```

Reply:
The previous version has a grammar error, so I'm trying to figure out how to correct it by arriving at mutually agreeable phrasing. :+1: 

Beyond that, we use restrictive clauses when writing about cases like this in the docs, so something like "pod configured with..." becomes something like "pod that is configured with..."

"the first pod that is configured to use the [dedicated?] network for pods"?

Reply:
ah ok, thanks for catching that. Yes, the latest version works for me:
`the first pod that is configured to use the dedicated network for pods`

Reply:
```suggestion
* The `enablePortPoolsPrepopulation` parameter controls pool prepopulation, which forces Kuryr to add ports to the pools when the first pod that is configured to use the dedicated network for pods is created in a namespace. The default value is `false`.
```